src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -1863,9 +1865,9 @@ void LedgerMasterImp::gotFetchPack (
     // InboundLedgers::gotFetchPack being called more than once
     // which is expensive. A flag should track whether we've already dispatched
 
-    getApp().getJobQueue().addJob (
+    app_.getJobQueue().addJob (
         jtLEDGER_DATA, "gotFetchPack",
-        [] (Job&) { getApp().getInboundLedgers().gotFetchPack(); });
+        [&] (Job&) { app_.getInboundLedgers().gotFetchPack(); });
```

Comment:
This looks like it could potentially have the same lifetime problem with the lambda. Less likely since `LedgerMasterImp` has a long lifetime.